test/unit/adapters/http.js

```diff
@@ -235,9 +232,10 @@ describe('supports http with nodejs', function () {
     }).listen(4444, function () {
       axios.get('http://localhost:4444/', {
         maxRedirects: 3
-      }).catch(function (error) {
+      }).catch(function(error) {
+        assert.equal(error.code, 'ERR_FR_TOO_MANY_REDIRECTS');
```

Comment:
More accurate, so that this test doesn't pass when any error happens, it passes when _that_ error happens specifically.